### PR TITLE
Add bubblesort program

### PR DESCRIPTION
**Pull request by tianyouw**
Also adds memcpy() to stdlib.c.
https://github.com/tianyouw/minRISC/pull/4

### CODE REVIEW

**Review by Denton-L — APPROVED** [review]

File: projects/briey/bubblesort/src/main.c [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+           iter++;
+        }
+    }
+
+    printf("Bubblesort complete after %d iterations.\r\n", iter);
+}
+
+void printArray(uint32_t array[], uint32_t size) {
+    printf("[%d", array[0]);
+    for (uint32_t i = 1; i < SORT_SIZE; i++) {
```
Comment (JYYao) [inline projects/briey/bubblesort/src/main.c]: Should this be size instead of SORT_SIZE?

**Review by JYYao — APPROVED** [review]

File: projects/briey/bubblesort/src/main.c (hunk 1 above)
Comment (tianyouw) [reply projects/briey/bubblesort/src/main.c]: Good catch, I'll change that.

File: projects/briey/memtest/makefile [hunk 2]
```diff
@@ -5,11 +5,7 @@ BENCH=no
 MULDIV=no
 # "yes" enables tree mem test and tree zeroing
 # run "make tree=no" to disable this
-TREE=yes
-
-ifeq ($(TREE),no)
-	CFLAGS += -DNO_TREE_TEST -DNO_TREE_INIT
-endif
+TREE=no
```
Comment (Jiangyi) [inline projects/briey/memtest/makefile]: Why did we change this to no?

Comment (tianyouw) [reply projects/briey/memtest/makefile]: I forgot to set it back after testing, changed it back to yes.

**Pull request merged**